iAM-legacy/RoadCare3/FormAnalysis.cs

```diff
@@ -1599,5 +1607,27 @@ private void DataGridViewRemainLife_CellMouseDoubleClick(object sender, DataGrid
                 }
             }
         }
+
+        private void CheckBoxReasons_CheckedChanged(object sender, EventArgs e)
+        {
+            var update = "";
+            if (checkBoxReasons.Checked)
+            {
+                update = "UPDATE SIMULATIONS SET USE_REASONS='1' WHERE SIMULATIONID='" + m_strSimulationID + "'";
```

Comment:
If we are using USE_REASON as a boolean, shouldn't we change it's type in the database?

Reply:
I believe this is a 1 or a 0 as the MSSQL database uses a "bit" field to handle true/false items. I don't think bool exists there.

Reply:
Agreed that BIT is the proper type here.

Reply:
Okay. I was thinking that USE_REASON is a new field added, and added as a string type rather than bit type.